firebase-firestore/src/main/java/com/google/firebase/firestore/model/value/ProtoValues.java

```diff
@@ -41,34 +41,47 @@
   public static final Value NULL_VALUE =
       Value.newBuilder().setNullValue(NullValue.NULL_VALUE).build();
 
+  /** The order of types in Firestore; this order is defined by the backend. */
+  private static final int TYPE_ORDER_NULL = 0;
```

Comment:
public methods that return private values make little sense. `typeOrder` should either be private or these constants should be public.

Reply:
Made them public.